src/main/java/wooteco/subway/domain/path/DiscountPolicy.java

```diff
@@ -0,0 +1,52 @@
+package wooteco.subway.domain.path;
+
+import static wooteco.subway.domain.path.DiscountPolicy.Constants.*;
+
+import java.util.Arrays;
+import java.util.function.Function;
+import java.util.function.Predicate;
+
+enum DiscountPolicy {
+
+    BABY_POLICY(age -> age < MIN_CHILD_AGE, fare -> 0),
+
+    CHILD_POLICY(age -> age >= MIN_CHILD_AGE && age < MIN_TEENAGER_AGE,
+        fare -> (int)((fare - DEFAULT_DEDUCTION) * CHILD_POLICY_RATE)),
+
+    TEENAGER_POLICY(age -> age >= MIN_TEENAGER_AGE && age < MIN_ADULT_AGE,
+        fare -> (int)((fare - DEFAULT_DEDUCTION) * TEENAGER_POLICY_RATE)),
```

Comment:
메서드로 추출할 수 있을 것 같아요 :)

Reply:
연령 비교 로직은 추출했을 때 더 가독성이 떨어진다고 생각하여 그대로 두고, 계산 관련 중복 로직을 메서드로 추출해보았습니다. 감사합니다👍
```suggestion
    CHILD_POLICY(age -> age >= MIN_CHILD_AGE && age < MIN_TEENAGER_AGE,
        fare -> calculateDiscountedFare(fare, CHILD_DISCOUNT_RATE)),

    TEENAGER_POLICY(age -> age >= MIN_TEENAGER_AGE && age < MIN_ADULT_AGE,
        fare -> calculateDiscountedFare(fare, TEENAGER_DISCOUNT_RATE)),
```